scripts/cloudfront_routing.js

```
@@ -0,0 +1,41 @@
+            "/about": "/about/index.html",
+            "/about/": "/about/index.html",
+            "/en": "/en/index.html",
+            "/en/": "/en/index.html",
+        };
+        
+        if (routeMap[uri] !== undefined) {
+            request.uri = routeMap[uri];
+            return request;
+        }
+        
+        if (uri.substr(-1) === '/') {
```

ESLint

[error] Expected indentation of 4 spaces but found 8.

(indent)